api/api.go

```diff
@@ -37,6 +39,43 @@ type SubmissionInfo struct {
 	State string `json:"state"`
 }
 
+// StatusInfo contains information about a user's status on a particular language track.
+type StatusInfo struct {
+	Language string `json:"track_id"`
```

Comment:
Yeah, let's do that in a separate commit. How about just making this one `TrackID` for now? (also, note: `TrackID`, not `TrackId`. See https://github.com/golang/go/wiki/CodeReviewComments#initialisms)